feat: adding json with unique (no duplicates) books inlcuding Kharos

elkolorado opened the pull request:
Making this so we have some standard when it comes to analysis of books. The books are sorted by length by default, so it is easier to reference to index of books when several people are working/sharing results.

s2ward commented:
Nice, we need something like this!  

I'm a bit confused by the naming, "kharos", does that include or exclude it? maybe a tad more descriptive filename regarding that. 

On another note, and maybe not in the scope of this PR:   
I was thinking if it would make sense to make the json a bit more verbose, e.g. list of objects or not so it's easier to point to them. could also give each book some properties, e.g. bookcase#, time of implementation.. lots of nice things :")  

```
{ hellgate : [{"id": "1", "content": "book", implemented, bookcase}, {"id": "2" .... }]} , 
kharos: [{}...]
isle_of_kings,
tibia_poll, 
tibia_doll, 
secret_library, 
honeminas, 
```
 
This would be quite nice as you can filter and work with all of them or a subset.  

Just food for thought, and this would require a bit more work and a tad more complexity when programmatically working with it. But would be a nice overview.  

Maybe if you rename the file i'll gladly approve it. good idea sorting by length

s2ward commented:
Oh with bookcase #, then we could show the duplicate book as well.

elkolorado commented:
yeah, makes sense, i've added `with` so it is more descriptive. About the properties of books - it is good idea to do it. But we need to discuss it further on what exactly they should have, and maybe on how to combat also it in structure that would also allow to include all 469 texts. We have books, we have npc talking bout it, data on website etc. For now I've also added in above fix /data folder so maybe in future we could extend it with various other jsons. For this PR I found this would work the best as is the most simple, and so in future when we reference to `book 22` we will know its kharos book or so.
